climada/hazard/test/test_tc_surge_bathtub.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, bounds, res_deg):
         res_deg : float
             Resolution in degrees
         """
-        lat = np.arange(bounds[1] + 0.5 * res_deg, bounds[3], res_deg)
+        lat = np.arange(bounds[3] - 0.5 * res_deg, bounds[1], -res_deg)
```

Comment:
While I was at it, I corrected the latitudinal orientation of the temporary topo data.